controllers/snapshot/snapshot_adapter.go

```diff
@@ -378,12 +378,12 @@ func (a *Adapter) EnsureSnapshotEnvironmentBindingExist() (reconciler.OperationR
 		} else {
 			snapshotEnvironmentBinding, err = a.createSnapshotEnvironmentBindingForSnapshot(a.application, &availableEnvironment, a.snapshot, components)
 			if err != nil {
-				a.logger.Error(err, "Failed to create SnapshotEnvironmentBinding",
-					"snapshotEnvironmentBinding.Application", snapshotEnvironmentBinding.Spec.Application,
-					"snapshotEnvironmentBinding.Environment", snapshotEnvironmentBinding.Spec.Environment,
-					"snapshotEnvironmentBinding.Snapshot", snapshotEnvironmentBinding.Spec.Snapshot)
+				a.logger.Error(err, "Failed to create SnapshotEnvironmentBinding for snapshot",
+					"application", a.application.Name,
```

Comment:
we log appocation name by default

Reply:
Thanks for the comment, I think `a.logger.Error()`  calls `logr.Logger` in package `github.com/go-logr/logr`  so will not log application, but our `a.logger.LogAuditEvent()` will log application.

Reply:
It should work, it's configured at the logger level, `LogAuditLevel` just calls `logger.Info()` and logs application details. There is no specfic code in that function to log application name

Reply:
I tried to log like an nil error like this
```
			a.logger.Error(err, "FFFFFFFFFailed to create SnapshotEnvironmentBinding for snapshot",
				"applicationnnnnnn", a.application.Name,
				"environment", availableEnvironment.Name,
				"snapshot", a.snapshot.Name)
```
and I can get log `ERROR <nil> FFFFFFFFFailed to create SnapshotEnvironmentBinding for snapshot applicationnnnnnn application-sample environment envname snapshot snapshot-sample`,  only one application name. What did I missed?

Reply:
According to https://github.com/redhat-appstudio/integration-service/blob/main/controllers/snapshot/snapshot_controller.go#L88, yes, we should log out the application name by default, but it doesn't, our adapter still use extra code to log the application name https://github.com/redhat-appstudio/integration-service/blob/main/controllers/snapshot/snapshot_adapter.go sometimes, we need to debug it

Reply:
I remember why.... this adapter doesn't call controller before creating adapter so that `logger.WithApp` is not called. Thanks! Let me update all these log code `logger.err` together to remote application.